docs/development/test-vectors.rst

```diff
@@ -332,6 +332,14 @@ Hashes
 * SHA2 (224, 256, 384, 512) from `NIST CAVP`_.
 * Whirlpool from the `Whirlpool website`_.
 
+Custom Hashes
```

Comment:
This section isn't required, you can just add a line after Whirlpool above that says


```
* Blake2s and Blake2b from OpenSSL `test/evptests.txt`_.
```